docs/UserGuide.md

```diff
@@ -76,75 +86,153 @@ Financial Assurance Revolutionary Telemarketer (F.A.R.T) is a **desktop app for
+  * The client's profile picture, highlighted by the purple box. This can be updated via the `upload` command.
+  * The client's details, highlighted by the orange box. This can be updated by various commands, including `edit`, `paid,` `unpaid`.
+  * Three template messages, highlighted by the green boxes. These are predetermined messages that have been categorised by the different life phases that a client could be in. Clicking on one of the buttons will copy the relevant message to your clipboard, allowing you to quickly send a message to your client via your preferred messaging app. The message will automatically use the **name provided in the F.A.R.T. book**. A successful copy will result in `Template Message copied to clipboard!` being shown in the `Response Window`.
 
-### Adding a client: `add`
+<div markdown="block" class="alert alert-info">
+  **:information_source: Note!**<br>
+The displayed contact will persist and edits will not be reflected until you select another contact or close the F.A.R.T book.If you made any changes to the current contact on display, kindly double-click on the same contact to refresh the detailed view.
+</div> 
+
+--------------------------------------------------------------------------------------------------------------------
+
+ ### Adding a client: `add`
 
 Met a potential client or someone new? This command adds a client to the F.A.R.T book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [a/ADDRESS] b/BIRTHDAY [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL b/BIRTHDAY [a/ADDRESS] [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A client can have any number of tags (including 0) but only maximum of 1 address
 </div>
 
 Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 b/11 09 2001`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 b/11 09 2001 t/criminal`
+* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/8234567 b/11 09 2001 t/criminal`
 
 **Note:**
-1. As of version 1.5, only valid email address (in the form name@domain.com) and **Singapore** phone numbers are accepted.
-Birthdays should be added in DD MM YYYY format.
-2. You may add duplicate contacts (i.e., contacts with the same name, case-insensitive) but the F.A.R.T book will prompt you
-to confirm that you are adding a duplicate contact.
+1. As mentioned in the Validation Rules, only valid email addresses (in the form `local-part@domain`) and **Singapore** phone numbers are accepted.
+1. Birthdays should be added in DD MM YYYY format.
+1. You may add duplicate contacts (i.e., contacts with the same name, case-insensitive) but the F.A.R.T book will prompt you to confirm that you are adding a duplicate contact. If you select `No` in the prompt window, the operation will be aborted and the client will not be added. Else, select `Yes` to add the new client.
```

Comment:
same index?

Reply:
Yes, will be numbered accordingly when converting.